config/deploy/docker/lib/memory_analyzer.rb

```diff
@@ -48,34 +52,52 @@ def run!
       return
     end
 
-    unless _ram_settings_havent_changed_in_two_weeks?
-      puts "[INFO][MEMORY_ANALYZER] Cannot analyze RAM as it has changed in the past two weeks or there's not enough history"
-      self.recommended_hard_limit_mb = (current_values.current_hard_limit_mb || scheduled_hard_limit_mb).to_i
-      self.recommended_soft_limit_mb = (current_values.current_soft_limit_mb || scheduled_soft_limit_mb).to_i
+    unless _ram_settings_havent_changed_recently?
+      puts "[INFO][MEMORY_ANALYZER] Cannot analyze RAM as it has changed in the past day or there's not enough history"
+      self.recommended_hard_limit_mb = scheduled_hard_limit_mb
```

Comment:
is `scheduled_hard_limit_mb` now stored somewhere?  You renamed the attr_accessor above.